streams/src/test/java/org/apache/kafka/streams/state/internals/FilteredCacheIteratorTest.java

```diff
@@ -47,7 +52,7 @@ public Bytes cacheKey(final Bytes key) {
     };
 
     @SuppressWarnings("unchecked")
-    private final InMemoryKeyValueStore<Bytes, LRUCacheEntry> store = new InMemoryKeyValueStore("name", null, null);
+    private final KeyValueStoreWrapper<Bytes, LRUCacheEntry> store = new KeyValueStoreWrapper<>();
```

Comment:
I felt it was reasonable here to simply wrap a plain old Map with the required functionality, rather than take advantage of the KeyValueBytesStoreWrapper used elsewhere. This was largely to avoid having to write serdes for LRUCacheEntry, but if anyone feels strongly about it this test can be reworked to follow the others.

Reply:
I think it's perfectly reasonable (and preferable) to use a simple map store so that we can more effectively isolate our testing concerns.

The only thing I'd add is that, even though it wasn't the intent of these other tests to exercise the bytes store, they did. So, we should make sure that the bytes store still has full coverage after these changes.

Reply:
For this specific test though, `FilteredCacheIterator` is only for the caching layer not for the underlying (in-memory) store layer.